.dockerignore

```diff
@@ -0,0 +1,14 @@
+.gitignore
+Dockerfile
+.github
+.pytest_cache
+/tests
+.coverage
+.flake8
+.gitignore
+coverage.xml
```

Comment:
The `.pytest_cache`, `.coverage` and `coverage.xml` files should be gitignore'd so not needed here.

Reply:
I must've committed these files before adding the .gitignore, I'll remove them from here. 👍